ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,41 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.isRadioGroup 'columns'}}" data-test-radio-input>
```

Comment:
Sort of thinking out loud about how we want the forms to work in the future -- I'm not sure the column format should be configured via "radio group", it's more of a horizontal vs vertical alignment setting. This could mean that for now we just update the variable name to `columnRadio` or something, and we assume that possibleValues with helperText should not be in columns. In the future it might be something we define explicitly on the attribute but we don't necessarily have to do that here

Reply:
Good point, Chelsea! I remember you brought this or something similar up during standup. I'll update the name of this getter and when this is revamped we can make this more general.